src/configs/env.ts

```diff
@@ -4,6 +4,10 @@
 export const WEBSITE_NAME = process.env.REACT_APP_WEBSITE_NAME || 'Reveal';
 export type WEBSITE_NAME = typeof WEBSITE_NAME;
 
+/** The website language */
+export const LANGUAGE = process.env.REACT_APP_LANGUAGE || 'en';
+export type LANGUAGE = typeof LANGUAGE;
```

Comment:
If no localization language is specified the app will use English translated display strings.